Source/Encoding/AOMEnc.vb

```diff
@@ -691,12 +698,12 @@ Public Class AV1Params
             If Not IsCustom(pass, "--cq-level") Then
                 sb.Append(" --cq-level=" & CqLevel.Value)
             End If
-        Else
+        End If
+
+        If Not RateMode.ValueText.Equals("q") Then
             If Not IsCustom(pass, "--target-bitrate") Then
-                If TargetBitrate.Value <> 0 AndAlso pass = 1 Then
```

Comment:
I couldn't think of a reason why you would want this if statement, if there is some reason I'm not aware of feel free to let me know and I'll revert this.